### PR TITLE
Issue 27358

### LINKED ISSUE #27358: Advance search page allow minus price filter
### Preconditions (*)

Install Magento 2.3.4

### Steps to reproduce (*)

1. Go to the Advance search page.
2. Add minus value in the price box

Screenshot from 2020-03-19 18-56-52

### Expected result (*)

It should return the validation error message: "Please enter a number 0 or greater in this field."

### Actual result (*)

It allows the minus value in search.

### PR DESCRIPTION
**Pull request by RHapani**
### Description (*)
The fixed issue of the Advance search page allow minus price filter.

### Steps to reproduce (*)
1. Go to the Advance search page.
2. Add minus value in price filter and search.

### Expected result (*)
It should not allow minus value in price filter.

### Actual result (*)
It allows the minus value in price filter.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#27358: Advance search page allow minus price filter

### Manual testing scenarios (*)
1. Go to the Advance search page.
2. Add minus value in price filter and search.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Catalog/Model/Category/Attribute/Backend/Image.php [hunk 1]
```diff
@@ -196,8 +196,9 @@ private function fileResidesOutsideCategoryDir($value)
         if (!$baseMediaDir) {
             return false;
         }
```
Comment (diazwatson) [inline app/code/Magento/Catalog/Model/Category/Attribute/Backend/Image.php]: @RHapani I don't think this change is necessary, could you please revert it?

**Review by diazwatson — CHANGES_REQUESTED** [review]

**Comment by RHapani** [comment]
@diazwatson I have updated the changes as per request.

**Review by diazwatson — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @diazwatson, thank you for the review.
[ENGCOM-7222](https://jira.corp.magento.com/browse/ENGCOM-7222) has been created to process this Pull Request
:eight_spoked_asterisk: @diazwatson, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Before:**
Screenshot from 2020-04-06 10-40-59

**After:**
screenshotsearch

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**